infrastructure/src/main/kotlin/com/dobby/external/prompt/ExperimentPostKeywordMapper.kt

```
@@ -0,0 +1,84 @@
+                try {
+                    MatchType.valueOf(matchTypeStr)
+                } catch (e: IllegalArgumentException) {
+                    MatchType.ALL
+                }
+            } ?: MatchType.ALL,
+            reward = dto.reward ?: "",
+            count = dto.count ?: 0,
+            timeRequired = dto.timeRequired?.takeIf { it.isNotBlank() }?.let { timeSlotStr ->
+                try {
+                    TimeSlot.valueOf(timeSlotStr)
+                } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)